src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Well, we get all of our existing coverage for non-brokered locations. These are particularly intrusive tests as we have to write to user folders. They aren't going to be rock solid either due to AV activity. I'm nervous about playing in the Music folder as it is- I figured that location was the least likely to cause permanent damage to user's machines. If we accidentally delete Pictures that would be especially bad, for example.

I do plan to add more, however. I just haven't had enough time as of yet.